### ISSUE TITLE
Add redux-form version to documentations

### ISSUE THREAD

**Issue by LAITONEN** [erikras/redux-form#3841]
### Are you submitting a **bug report** or a **feature request**?

Feature request.

### What is the current behavior?

The documentation pages have no indication of the version of Redux-form, for which the documentation is written. I spent more than an hour reading the documentation of older version (6.0.0alpha), which I got redirected to from Google. And since I am using Safari, the url path, which has the version of redux-form indicated, is hidden.
<img width="1280" alt="screen shot 2018-02-17 at 2 38 18 pm" src="https://user-images.githubusercontent.com/23742868/36341018-5d69d4bc-13f0-11e8-8270-4dfea49d8077.png">

### What is the expected behavior?
To have version number visible on any page of documentation. Usually, it is added to the header: left or right corner. For that you can check:
[Lodash](https://lodash.com/docs/)
[React](https://reactjs.org)

### What's your environment?

Browser: Safari 10.1.2

**Comment by gustavohenke** [comment]
Like this?
https://github.com/erikras/redux-form/issues/3397#issuecomment-328570882

As you will notice, v6.0.0-alpha.4 is the only problematic version, and it insists in being among the first results on Google.

**Comment by LAITONEN** [comment]
@gustavohenke I was more talking about adding a version number (as an image / dropdown / text) to the website and, optionally, possibility to interactively switch the version of the documentation. Examples are Lodash and React websites, which I linked above.

**Comment by gustavohenke** [comment]
Yes, I'm aware of what you mention, but the said warning banner _should work_ to let users know they are browsing the wrong version.

You can also change the docs version as well, but it's quite hidden at the moment:  
https://redux-form.com/7.2.3/docs/documentationversions.md/

**Comment by LAITONEN** [comment]
@gustavohenke Well, that's great to know that the team is aware of the issue. Will be waiting for the solution. Thanks!

**Comment by piedrahitapablo** [comment]
This issue is still present on older docs versions, there is no warning or sign that you are browsing an older version.
